build/pipelines/templates/run-staticAnalysis.yaml

```diff
@@ -5,11 +5,11 @@
 
 steps:
   - powershell: |
-      Install-Module -Name PSScriptAnalyzer -Repository PSGallery -Scope CurrentUser -AllowClobber -SkipPublisherCheck -Force -Verbose
+      Install-Module -Name PSScriptAnalyzer -Repository PSGallery -Scope CurrentUser -Verbose
```

Comment:
```suggestion
      Install-Module -Name PSScriptAnalyzer -Repository PSGallery -Scope CurrentUser -AllowClobber -SkipPublisherCheck -Force -Verbose
```

Removing those parameters causes it to not install.  From a [CI build with that change](https://dev.azure.com/ms/PowerShellForGitHub/_build/results?buildId=87771&view=logs&j=2d2b3007-3c5c-5840-9bb0-2b1ea49925f3&t=ac0b0b0f-051f-52f7-8fb3-a7e384b0dde9):

```
VERBOSE: Repository details, Name = 'PSGallery', Location = 'https://www.powershellgallery.com/api/v2'; IsTrusted = 'False'; IsRegistered = 'True'.
VERBOSE: Using the provider 'PowerShellGet' for searching packages.
VERBOSE: Using the specified source names : 'PSGallery'.
VERBOSE: Getting the provider object for the PackageManagement Provider 'NuGet'.
VERBOSE: The specified Location is 'https://www.powershellgallery.com/api/v2' and PackageManagementProvider is 'NuGet'.
VERBOSE: Searching repository 'https://www.powershellgallery.com/api/v2/FindPackagesById()?id='PSScriptAnalyzer'' for ''.
VERBOSE: Total package yield:'1' for the specified package 'PSScriptAnalyzer'.
VERBOSE: Performing the operation "Install-Module" on target "Version '1.19.0' of module 'PSScriptAnalyzer'".
WARNING: User declined to install module (PSScriptAnalyzer).
```

Note the last line.  That's what causes the failure in the [next step](https://dev.azure.com/ms/PowerShellForGitHub/_build/results?buildId=87771&view=logs&j=2d2b3007-3c5c-5840-9bb0-2b1ea49925f3&t=e0810b27-de1e-5f7f-0df7-1a1042e925c2):

```
Invoke-ScriptAnalyzer : The term 'Invoke-ScriptAnalyzer' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At D:\a\_temp\c7dc4a6f-d3c1-48c6-bea2-b0264e544984.ps1:3 char:18
+ $results = try { Invoke-ScriptAnalyzer -Path ./ –Recurse -ErrorAction …
+                  ~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : ObjectNotFound: (Invoke-ScriptAnalyzer:String) [], ParentContainsErrorRecordException
+ FullyQualifiedErrorId : CommandNotFoundException
##[error]PowerShell exited with code '1'.
Finishing: Run Static Code Analysis (PSScriptAnalyzer)
```

Reply:
Aha, that seems to be an old PS5 legacy of PowerShellGet. The only switch you need is `-Force` then. Skipping the publisher check is bad from a security perspective

Reply:
Actually, go ahead and make that change in this one and i'll kick CI off a few more times looking for the repro.  Since you've kept it as a PS5 task, it should be safe to merge into master as well.

Reply:
Ok, done